benchmark/utils/CxxTests/CreateObjects.h

```diff
@@ -0,0 +1,3 @@
+struct CxxIntBox {
```

Comment:
"CxxIntWrapper" b/c the int is not on the heap?

Reply:
SGTM.